bin/oref0-setup.sh

```diff
@@ -754,7 +770,7 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
     # TODO: deprecate g4-upload and g4-local-only
     if [[ ${CGM,,} =~ "g4-upload" ]]; then
         mkdir -p $directory-cgm-loop
-        if ( cd $directory-cgm-loop && ls openaps.ini 2>/dev/null >/dev/null && openaps use -h >/dev/null ); then
+        if ( cd $directory-cgm-loop && ls openaps.ini &>/dev/null && openaps use -h >/dev/null ); then
```

Comment:
As above, the intention here is to squash "file not found" errors.

Reply:
yep so instead of writing `2>/dev/null >/dev/null`, can be simplified to `&> /dev/null` in bash

Reply:
Ah, ok.  I wasn't familiar with that syntax.  These are all good then, thx.